new_module.py

```diff
@@ -0,0 +1,13 @@
+class Genius:
+    def __int__(self):
+        self._memory = -1
```

Comment:
show type of this var, and init it with none or some class method if you don't take a value as init arg